dotcom-rendering/src/web/components/ArticleTitle.tsx

```diff
@@ -65,30 +77,43 @@ export const ArticleTitle = ({
 	guardianBaseURL,
 	badge,
 	isMatch,
-}: Props) => (
-	<div css={[sectionStyles, badge && badgeContainer]}>
-		{badge && format.display !== ArticleDisplay.Immersive && (
-			<div css={titleBadgeWrapper}>
-				<Badge imageUrl={badge.imageUrl} seriesTag={badge.seriesTag} />
+}: Props) => {
+	const palette = decidePalette(format);
+	return (
+		<div css={[sectionStyles, badge && badgeContainer]}>
+			{format.design === ArticleDesign.LiveBlog && (
+				<span css={livePulseIconStyles(palette)}>
+					<Island deferUntil="idle">
+						<PulsingDot diameter={1.084} />
```

Comment:
If this diameter is always the same, is there a reason to pass it in as a prop?

Reply:
The `PulsingDot` is used in 3 different places and it's size is different in every one of them. I thought we can do either of these:
1- add a context prop such as  `kicker | ArticleLastUpdated | ArticleTitle` to `PulsingDot` to decide the size based on the context
2- Pass the size of the dot as a prop

But wasn't sure which option to go for 🤔   @oliverlloyd What's your insight about this?

Reply:
I think those both sound like valid options.

I think though given the dot is always shown alongside a kicker then if we had a `size` prop we could make sure that each size aligns with it's own text `size` value.

Responding to you other comment below:
The design system uses `xxxsmall`, `xxsmall`, etc. which we _could_ use but, yeah, equally we could also just have `small`|`medium`|`large. I'd perhaps be wary of coupling things _too_ close to the design system as that will change and then things will be confusing so maybe `small`, `medium`, etc. is simplest. We use similar apis in other places

Reply:
I don’t think we need to add a `size` prop at all! The icon is already sized relative to current `font-size`, as it uses `em`–not `rem`.

What happens in you remove the `size`/`diameter` prop altogether?

Reply:
FYI I’ve got it to work by removing the font-size style entirely and the diameter:

<img width="131" alt="image" src="https://user-images.githubusercontent.com/76776/164221947-79fbf7e3-e1f6-49f9-8882-e3835db37bf3.png">

<img width="351" alt="image" src="https://user-images.githubusercontent.com/76776/164222006-ba51d15d-5e15-4202-b96d-31f433f19913.png">

Reply:
> FYI I’ve got it to work by removing the font-size style entirely and the diameter

Woo hoo 🎉

Reply:
@marjisound and I discussed it and the reason for introducing this was that the Figma design specified a 11px dot for a font-size of 15px, which is just shy of `0.75em`. This is probably not reason enough to introduce a whole diameter parameter, but we could reduce to `0.74em` to get the right rounding of values for specific font-sizes.

Reply:
Having a chat with Ben he was happy with us using the `0.75em` meaning the dot diameter would be 75% of the size of the text beside it. 

I moved the usage of the `PulsingDot` from the `ArticleTitle` to the `SeriesSectionLink` mostly because in SeriesSectionLink, we have the font styles and the colour which is needed by the pulsing dot